### PR TITLE
Add pmysql-dsb support with tests

### PR DESCRIPTION
**Pull request by vjtirumalai**
- [x] Local build passes
- [x] Functional tests passed with 0.23 
- [x] Works with CF deployer and shpanCRB 0.24

Defer to next iteration:
- [ ] Make functional tests work with CRB FS version based on 0.24
- [ ] CI pending

### CODE REVIEW

File: pom.xml
Comment (shpandrak) [inline pom.xml]: There is no need for this anymore, lets talk

File: pom.xml
Comment (shpandrak) [inline pom.xml:1]: Please remove this entire pom and flatten the project to have only the mysql-dsb as root pom. this pom adds nothing.. lets talk

File: pom.xml
Comment (shpandrak) [inline pom.xml]: the groupId of utilities is incorrect. please use:
    <groupId>com.emc.ocopea.util</groupId>

File: pom.xml
Comment (shpandrak) [inline pom.xml]: lets talk about component version and how we manage them, I want it to be the same across the projects

File: mysqldsb/pom.xml
Comment (shpandrak) [inline mysqldsb/pom.xml]: This pom should be the projects root pom..

File: mysqldsb/pom.xml
Comment (shpandrak) [inline mysqldsb/pom.xml]: version should be snapshot, but lets talk...

File: mysqldsb/service/src/main/java/com/emc/nazgul/dsb/mysql/MySqlDSBMicroService.java
Comment (shpandrak) [inline mysqldsb/service/src/main/java/com/emc/nazgul/dsb/mysql/MySqlDSBMicroService.java]: This appears in the API, no need to mention it is in java :) should say something about PCF of p-mysql

File: mysqldsb/service/src/main/java/com/emc/nazgul/dsb/mysql/MySqlDSBMicroService.java
Comment (shpandrak) [inline mysqldsb/service/src/main/java/com/emc/nazgul/dsb/mysql/MySqlDSBMicroService.java]: The package directory structure does not match the package name (there is nazgul in the folder name...)

File: pom.xml
Comment (vjtirumalai) [reply pom.xml:1]: Thanks for fixing

**Comment by vjtirumalai** [comment]
Will wait for CRB FS side changes before this gets merged.

File: mysqldsb/service/src/main/java/com/emc/nazgul/dsb/mysql/MySqlDSBMicroService.java
Comment (vjtirumalai) [reply mysqldsb/service/src/main/java/com/emc/nazgul/dsb/mysql/MySqlDSBMicroService.java]: Thanks for fixing.

File: pom.xml
Comment (shpandrak) [reply pom.xml:1]: At your service

File: pom.xml
Comment (vjtirumalai) [reply pom.xml]: Thanks for fixing.

File: pom.xml
Comment (vjtirumalai) [reply pom.xml]: Fixed by removing.

File: pom.xml
Comment (vjtirumalai) [reply pom.xml]: sure next iteration.

File: mysqldsb/pom.xml
Comment (vjtirumalai) [reply mysqldsb/pom.xml]: Thanks for fixing

File: mysqldsb/pom.xml
Comment (vjtirumalai) [reply mysqldsb/pom.xml]: Thanks for fixing.

File: mysqldsb/service/src/main/java/com/emc/nazgul/dsb/mysql/MySqlDSBMicroService.java
Comment (vjtirumalai) [reply mysqldsb/service/src/main/java/com/emc/nazgul/dsb/mysql/MySqlDSBMicroService.java]: Good point. Fixed.

**Review by shpandrak — APPROVED** [review]

**Pull request merged**
